lms/djangoapps/certificates/apis/v0/tests/test_views.py

```diff
@@ -0,0 +1,107 @@
+"""
+Tests for the Certificate REST APIs.
+"""
+import json
+
+from django.core.urlresolvers import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from certificates.tests.factories import GeneratedCertificateFactory
+from certificates.models import CertificateStatuses
+
+from student.tests.factories import UserFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+
+class CertificatesRestApiTest(ModuleStoreTestCase, APITestCase):
+    """
+    Test for the Certificates REST APIs
+    """
+
+    def setUp(self):
+        super(CertificatesRestApiTest, self).setUp()
+
+        self.student = UserFactory(password='test')
+        self.student_no_cert = UserFactory(password='test')
```

Comment:
Can you use `UserFactory.create` for consistency with how `CourseFactory` and `GeneratedCertificateFactory` are used below? (or the other way around)